### DISCUSSION TITLE
Creating unconventional graph (G=(U,V))

### DISCUSSION THREAD

**Discussion by phiofi** [3b1b/manim#1548] [Q&A] [answered]
I am discovering the class ```Graph``` in ```manim```. Long story short, I am trying to generate the so-called "bat" from semigroup theory:
[bat][1]

With help of the documentation, I have ended up here:
[my_bat][2]

My questions are related to making the above pictures slightly more similar:
 1. How could I make an edge so that it will be dashed? 
 2. Is it possible for an edge to be round as in a cycle starting in *a_n*?
My code for now:
```from manim import *

class Monogenic(Scene):
    def construct(self):
        vertices = [1, 2, 3, 4, 5, 6, 7, 8]
        edges = [(1, 2), (2, 3), (3, 4), (4, 5), (5,6), (6,7), (7,8), (8,5)]
        lt = {1: [0, 0, 0], 2: [1, 0, 0], 3: [2, 0, 0], 4: [3, 0, 0], 5: [4, 0, 0], 6:[5,1,0], 7:[6, 0, 0], 8:[5,-1,0]}
        G = Graph(vertices, edges, layout=lt,  edge_config={(4, 5): {"stroke_color": RED},
                               (7, 8): {"stroke_color": RED}})
        self.add(G)
```

Thanks in advance.

  [1]: https://i.stack.imgur.com/K2iRe.png
  [2]: https://i.stack.imgur.com/yXVUe.png

**Comment by TonyCrane** [comment, marked as answer]
I think you are using the community version. So please ask this question on there discord channel or open an issue in [ManimCommunity/manim](https://github.com/ManimCommunity/manim), instead of here.
